src/OSPSuite.InstallationValidator.Core/Assets/Constants.cs

```diff
@@ -1,13 +1,36 @@
-﻿namespace OSPSuite.InstallationValidator.Core.Assets
+﻿using System.Text;
+
+namespace OSPSuite.InstallationValidator.Core.Assets
 {
    public static class Constants
    {
-      public class Captions
+      public static class Captions
       {
          public static readonly string OutputFolder = "Output Folder";
          public static readonly string SelectOutputFolder = "Select Output Folder";
          public static readonly string Start = "Start";
          public static string TheValidationWasCanceled = "The validation was canceled";
+
+         public static class Exceptions
+         {
+            public static string ExceptionSupportMessage(string issueTrackerUrl)
```

Comment:
Because the message contains embedded html tags and assumes that there's a 'copy to clipboard' button available. That message had to change a bit to suit the situation here.